main.js

```diff
@@ -7,32 +7,68 @@ document.querySelector("#app").innerHTML = `
     <input id="location" placeholder="enter an airport..." />
     <button id="submit">enter</button>
   </div>
-  <div id="results"></div>
+  <div id="results">
+    <city-card location="Atlanta, Georgia, United States" website="http://www.atl.com/" image="https://i.ibb.co/52zkWYD/top-otolaryngology-facial-plastic-surgery-doctors-atlanta-georgia-usa.jpg"></city-card>
+  </div>
   <template id="output-template">
     <style>
-        img {
-          width: 50%;
+        .text {
+          color: black;
+        }
+        #image {
+          width: 20rem;
+        }
+        #radar-image {
+          width: 3rem;
+          position: absolute;
+          left: 0px;
+          top: 5px;
         }
         #city-output {
           background-color: rgb(64, 64, 64);
           border-radius: 20px;
-          display: flex;
-          flex-direction: column;
-          justify-content: center;
-          align-items: center;
           min-height: 30rem;
           min-width: 25rem;
           padding: 2rem;
+          position: relative;
+          text-align: center;
+          font-weight: bold;
+          border: 3px solid red;
         }
         #heading{
-          font-weight: bold;
           font-size: 20px;
           position: relative;
-          top: 0;
+          border: 3px solid purple;
+        }
+        #main-section{
+          height: 100%;
+          position: relative;
+          display: flex;
+          flex-direction: row;
+          padding: 2rem;
+          border: 3px solid green;
         }
-        #website{
+        #attractions, #info{
+          position: relative;
+          width: 20rem;
+          height: 25rem;
+          border: 3px solid yellow;
+        }
+        #next-flight{
+          position: relative;
+          width: 99%;
```

Comment:
```suggestion
          width: 100%;
          box-sizing: border-box;
```

Looking at this, I think the problem you were having that made this 99% is your [box model](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model). By switching to `border-box` you can have this at 100% without the border width changing the width